scripts/kokoro.sh

```diff
@@ -48,28 +49,11 @@ BAZEL_VERSION=0.23.1
 wget https://github.com/bazelbuild/bazel/releases/download/"${BAZEL_VERSION}"/bazel-"${BAZEL_VERSION}"-installer-linux-x86_64.sh
 chmod +x ./bazel-"${BAZEL_VERSION}"-installer-linux-x86_64.sh
 ./bazel-"${BAZEL_VERSION}"-installer-linux-x86_64.sh --user
-$HOME/bin/bazel version
-
-# Run the bazel test command multiple times because of some flaky
-# behavior with libssl.so
-tries=15
-set +e
-for ((i=1;i<=$tries;i++));
-do
-  $HOME/bin/bazel clean
-  $HOME/bin/bazel test --cache_test_results=no \
-    //javascript/net/grpc/web/... \
-    //net/grpc/gateway/examples/...
-  ret=$?
-  if [ $ret -eq 0 ]; then
-    break
-  elif [ $i -eq $tries ]; then
-    exit 1
-  fi
-done
-set -e
-rm ./buildifier
 rm ./bazel-"${BAZEL_VERSION}"-installer-linux-x86_64.sh
+$HOME/bin/bazel version
+$HOME/bin/bazel test \
```

Comment:
Can you add back the `$HOME/bin/bazel clean` line please? It makes running the script locally and repeatedly a little more robust. thanks.

Reply:
Sure. Done.